migration-engine/connectors/sql-migration-connector/src/sql_imperative_migration_persistence.rs

```diff
@@ -44,7 +44,7 @@ impl ImperativeMigrationsPersistence for SqlMigrationConnector {
         let id = Uuid::new_v4().to_string();
         let now = chrono::Utc::now();
 
-        let insert = Insert::single_into(IMPERATIVE_MIGRATIONS_TABLE_NAME)
+        let insert = Insert::single_into((self.schema_name(), IMPERATIVE_MIGRATIONS_TABLE_NAME))
```

Comment:
It's technically ok, it should work on other databases, but we moved away from explicit schema names everywhere else. I don't think it's a blocker though, i'll let you decide if we leave it like that, or we have a `table_name(name: &str) -> quaint::ast::Table<'_>` method on `SqlFlavour` where we add the schema name for mssql and not the other dbs.

Reply:
Fixed in https://github.com/prisma/prisma-engines/pull/1372/commits/773ad9ea1f1c46a287442113a97c1f99e22755c6